src/entity/Human.php

```diff
@@ -211,6 +211,14 @@ public function getXpManager() : ExperienceManager{
 		return $this->xpManager;
 	}
 
+	public function getXpSeed() : int{
```

Comment:
I understand, it's just vanilla naming. Then ideally the $xpSeed property should be renamed to $enchantmentSeed as well. But the TAG_XP_SEED will remain the same? 
https://github.com/pmmp/PocketMine-MP/blob/1ed7559eb1ea9a7e029b9c4f7f88637a543efa7e/src/entity/Human.php#L94C49-L94C49

Reply:
TAG_XP_SEED is merely intended to reflect vanilla save data. I agree with Ivan that this should be called "enchantment seed" instead.

We can't rename the property since it's `protected`, but the APIs can have better names.